packages/framework/src/Component/Cron/CronModule.php

```diff
@@ -133,6 +133,14 @@ public function isEnabled(): bool
         return $this->enabled;
     }
 
+    /**
+     * @param bool $enabled
+     */
+    public function setEnabled(bool $enabled): void
```

Comment:
I woulld prefer to remove the parameter and implement `setDisabled()`.

Reply:
or `enable()` and `disable()`